Formula/consul-backinator.rb

```diff
@@ -0,0 +1,33 @@
+class ConsulBackinator < Formula
+  desc "Consul backup and restoration application"
+  homepage "https://github.com/myENA/consul-backinator"
+  url "https://github.com/myENA/consul-backinator.git",
+    :tag => "v1.2",
+    :revision => "97c3d8506f332ba86b3af922cc20adb734c81dbc"
+  head "https://github.com/myENA/consul-backinator.git"
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
+
+    dir = buildpath/"src/github.com/myENA/consul-backinator"
+    dir.install buildpath.children
+
+    cd dir do
+      system "glide", "install", "-v"
+      system "go", "build", "-v", "-ldflags",
```

Comment:
You mean the next line that starts "-X ... " and the rest should line up with "go"?